src/node/rpc/frontend.h

```diff
@@ -345,6 +349,7 @@ namespace ccf
       {
         try
         {
+          pre_exec(tx, *ctx.get(), *this);
```

Comment:
So this pre-exec pattern may be needed (the "drop views on conflict" change is a bit dodgy, and we know it loses signatures too). But we believed it was safe for the PBFT case because we wouldn't get conflicts in the PBFT execution path - the order is pre-calculated/pre-executed to be non-conflicting? Can you explain why that assumption was wrong?

Reply:
On the primary we are producing the order so there are conflicts. This should not be a problem on the backup. Since there are conflicts we are dropping the part of the tx that writes the request.